### DISCUSSION TITLE
Multiple Ngx-editors not working under same component textarea is disabled unable to type something in textarea

### DISCUSSION THREAD

**Discussion by kpul** [sibiraj-s/ngx-editor#413] [Q&A]
I am working with Angular 9 version with Ngx-editor 9. In my component I Initialised ngx-editor as per documentation. In html page using multiple editors. If we open one editor,its  textarea is coming. when we close this editor and opened newone then second editor textarea not coming.
Please help me. I am providing stackblitz link where I modified code like in app.component.html same editor code paste then two editors displaying but first editor not showing anything.

https://stackblitz.com/edit/ngx-editor?file=src%2Fapp%2Fapp.component.ts

**Comment by sibiraj-s** [comment]
Make sure to check your destroy method. If you are to use two editors, I recommend to created two editor instance. To approach this. Move the editor and its initialization into an isolated component for scoping and use that component instead.
